File: dart/dart-sample/sample/collection/dice_game.dart
import 'dart:math';

void main(List<String> args) {
  const N = 10000000;
  var dice = {for (var i = 1; i <= 6; i++) i: 0};

  for (var i = 0; i < N; i++) {
    var n = Random().nextInt(6) + 1;
    dice[n] = dice[n]! + 1;
  }

  print(dice);
}

File: nodejs/javascript-sample/collection/dice-game.js
let dice = new Map()

const N = 100000

for (let i = 0; i < N; i++) {
  let key = Math.floor(Math.random() * 6 + 1)
  let value = dice.get(key) || 1
  dice.set(key, value + 1)
}

console.log(dice)
